### ISSUE TITLE
[EN][Class][Talent] Shatter

### ISSUE THREAD

**Issue by ExoDuSSS123** [Atlantiss/BugTracker#9693]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: 
![shatter](https://user-images.githubusercontent.com/22661211/33490700-4f616922-d6b8-11e7-8d15-0c485dae2d71.jpg)
**Shatter** should multiply my critical strike chance by x3 on frozen targets.
Combining that with **Fingers of Frost**
![fingers](https://user-images.githubusercontent.com/22661211/33490845-af0924aa-d6b8-11e7-8410-5e5d3fecc052.png)
should make my **Ice Lance** and **Deep Freeze** critical strike chance multiply by 3, whenever **Fingers of Frost** are on me
![postac](https://user-images.githubusercontent.com/22661211/33491262-f5b44064-d6b9-11e7-830e-ee62c2ea3f86.png)
Watching at my stats i have _**33,65%**_ crit chance with crit buff.
Now add to that Shatter talent, **_33,65% * 3 = 100.95% crit chance_**. I should have 100% crit chance on **Deep Freeze** + **Ice Lance**.

**Current behaviour**: 
![deep freeze](https://user-images.githubusercontent.com/22661211/33491461-a8440f2a-d6ba-11e7-8a61-c5de2f790095.png)
**Deep Freeze** can be used only on **Frozen** targets, soo I can use **Deep Freeze** only when **Fingers of Frost** buff is on me in raid conditions, because you can't make any boss Frozen. That should make **Deep Freeze** have 100% critical strike with my state of equipment.

The problem is that I usually end Boss combats with around 70% critical's on **Deep Freeze**.

![deep](https://user-images.githubusercontent.com/22661211/33493939-f62e762e-d6c1-11e7-9078-3e9f3a414fe0.png)
As you can see my overall critical strike chance on **Deep Freeze** from whole dungeon was:            <p>100% - 7/25*100=700/25=28%   =  **72% CRITS**   **28% HITS**</p>
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Just to be sure, I use my pet spell each time i cast **Deep Freeze**, so I always have 2stacks of **Fingers of Frost** on me during **Deep Freeze** cast.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

By the way, I had to test it on dungeon boss ofc with crit buff, becouse you can Freez dummys, and do 0dmg to them by **Deep Freeze** ( they should be immune to Freez effect's, I can't even test different rotation's on dummys cuz my 2nd damage dealing spell does 0dmg... )

**Expected behaviour**: I should end any combat with crit buff, having 100% crits on **Deep Freeze** and on **Ice Lance**.

**Steps to reproduce the problem**:

1. Get Frost mage with at least 33,34% crit chance to any mob that cannot be frozen.
2. Get **Fingers of Frost** buff by hitting with **Frostbolt**.
3. Use Deep Freeze several times.
4. Check your % of critical strikes on Deep Freeze

**Revision**: 5536

![deep bez lege](https://user-images.githubusercontent.com/22661211/33495797-1562a67c-d6c8-11e7-8dc1-a8ccabbcb50a.png)
to all those that will post that it's coused by Legendary. DEEP FREEZE WITHOUT LEGENDARY ON. still with 34% crit

**Comment by poleloa** [comment]
bugged skills exozus? 

postscirptum :nice gear. Gl for you.

**Comment by ExoDuSSS123** [comment]
![shadow pve](https://user-images.githubusercontent.com/22661211/33743016-5613b642-dbab-11e7-82b2-24c873387d25.png)
**Deep Freeze** from today's Baleroc  hc, with 2 stack's of Fingers Of Frost each time I used **Deep Freeze**, to prevent copied **Deep Freezes** from legendary to do not have Shatter...

**Comment by Piklo** [comment]
iirc talents are also listed in SPELLbook, and we don't have label like "talent", so "spell" is probably better than nothing :D

**Comment by Frostbreaker** [comment]
Can confirm, this is serious bug if someone acutally want to play frost mage pve, which is apparently very good not only in pvp. Hope this will be fixed soon.

**Comment by Olrios** [comment]
**All of you are wrong, deep freeze is blizz like here. Just learn how it works and how to use it:**

Deep Freeze REAL TEXT:
To use it on targets premanently immune to stun you need atleast one stack of finger of frost. Deals XXX damage splitted into two times to targets premanently immune to stun where each splitted damage's critical strike chance is multiplied by shatter only if is possible to consume one finger of frost per each split.

So, you use 1 deep freeze which deals 200k damage on 1 target where:
A) you have 1 fof = deep freeze deals two times damage, one 100% crit chance for 100k and 2nd 33% crit chance for ~50k (or 100k if crits)
B) you have 2 fof = deep freeze deals 2x damage with 100% crit chance for 100k+100k

You need 2 stacks of finger of frost to make deep freeze splits crit twice.
Everything works normal, deep freeze and fof is blizz like with on atlantiss.

**This topic should be closed.**

**Comment by poleloa** [comment]
hmm are u sure? greetings

2018-05-27 12:49 GMT+02:00 Olrios <notifications@github.com>:

**Comment by Olrios** [comment]
I'm sure problem is with legendary, which sometimes copies spells in the wrong way. In some cases 100k spell is copied (100k+100k) but in some cases spell is not copied but doubled (one hit for 200k). I'm sure bug is with legendary. I raided for over 20 hours without legendary yet and frost mage's DF worked fine, 100% blizz like. After few tests with guildmates we noticed many bugs with how legendary doubles/copies spells. When we had enought clean info, new topic will be added.
So, focus on tests w/o legendary and see that DF works 100% blizz like, or simply read some frost mage tutorials for help.
